internal/commands/list_buildpack_registries_test.go

```diff
@@ -76,11 +76,21 @@ func testListBuildpackRegistriesCommand(t *testing.T, when spec.G, it spec.S) {
 			h.AssertContains(t, outBuf.String(), "public registry")
 			h.AssertContains(t, outBuf.String(), "https://github.com/buildpacks/public-registry")
 
-			h.AssertContains(t, outBuf.String(), "private registry")
+			h.AssertContains(t, outBuf.String(), "* private registry")
 			h.AssertContains(t, outBuf.String(), "https://github.com/buildpacks/private-registry")
 
 			h.AssertContains(t, outBuf.String(), "personal registry")
 			h.AssertContains(t, outBuf.String(), "https://github.com/buildpacks/personal-registry")
 		})
+
+		it("should ALWAYS list an official default registry", func() {
```

Comment:
We should also add a test that "official" cannot be redeclared via `add-buildpack-registry`. We can discuss more intricate edge-cases later but for now we should allow for "official" to be listed twice.